### PR TITLE
update README with ardupilot configuration

### PR DESCRIPTION
**Pull request by pedro-fuoco**
Provides the user with the information required to send the odometry generated from the SLAM to Ardupilot's EKF, which was made possible by ArduPilot/ardupilot#24518

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,17 @@ With the world and copter in place, launch cartographer to generate SLAM:
...
+
+-  AHRS_EKF_TYPE = 3 to use EKF3
+-  EK2_ENABLE = 0 to disable EKF2
+-  EK3_ENABLE = 1 to enable EKF3
+-  EK3_SRC1_POSXY = 6 to set position horizontal source to ExternalNAV
+-  EK3_SRC1_POSZ = 1 to set position vertical source to Baro
+-  EK3_SRC1_VELXY = 6 to set velocity horizontal source to ExternalNAV
+-  EK3_SRC1_VELZ = 6 to set vertical velocity source to ExternalNAV
+-  EK3_SRC1_YAW = 6 to set yaw source to ExternalNAV
+-  GPS_TYPE = 0 to disable the GPS
```
Comment (tridge) [inline README.md]: why disable the GPS? On real vehicles it would be very helpful for log analysis, even if not actually used in flight

Comment (pedro-fuoco) [reply README.md]: Fair point! I will change it here and in ArduPilot/ardupilot_wiki#5366

**Comment by tridge** [comment]
worth linking to https://ardupilot.org/copter/docs/common-ekf-sources.html

File: README.md (hunk 1 above)
Comment (Ryanf55) [reply README.md]: Agree with Tridge that normally users, particularly researches, would have GPS available and have it as a source of truth, or at least another source like a Vicon. 

## Common setup
Primary: GPS
Secondary: External (from ROS)

And a switch to flip between the two. Both are normally recorded.

## Other common setup
EK2: GPS
EK3: External (from ROS)


## Plan

Given this is a safety problem, if cartographer fails, we should at least explain what the "safe" mode is. AltHold is probably ok. If Cartographer causes the EKF to diverge, Tridge would have the external as ride-along (log only, no control impact). Using RC7_OPTION see EKF_POS_SOURCE of 90.

**Comment by pedro-fuoco** [comment]
@tridge @Ryanf55 done!

**Review by tridge — APPROVED** [review]

**Pull request merged**